components/contribution-flow/index.js

```diff
@@ -720,16 +761,17 @@ class ContributionFlow extends React.Component {
                     taxes={this.getApplicableTaxes(collective, host, tier?.type)}
                     onSignInClick={() => this.setState({ showSignIn: true })}
                     isEmbed={isEmbed}
-                    isSubmitting={isValidating || isSubmitted || isSubmitting}
+                    isSubmitting={isValidating || isLoading}
+                    order={this.state.createdOrder}
                   />
                   <Box mt={40}>
                     <ContributionFlowButtons
                       goNext={goNext}
-                      goBack={goBack}
+                      goBack={isCrypto && currentStep.name === 'checkout' ? null : goBack} // for crypto flow the user should not be able to go back after the order is created at checkout step
```

Comment:
The comment should not go so far to the left, find something else

Reply:
It's a detail I know, not mandatory if it's a pain.

Reply:
Hmm, I've added the comment on top of the `goBack` prop. This way I think it looks better. 🤔